content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
```

Comment:
Using multi-network for VMs is not fully implemented or supported.

In all releases up to 1.6  L3 networking has been required - if somehow multi-network works (partially or not), we can document it  - but it can't be the default.

Reply:
This language was decided in https://github.com/istio/istio.io/pull/7248#discussion_r423283719. Feel free to offer up a gist of how this works.

Cheers
-steve

Reply:
I think it's critical to explain how VM makes call to the pod, and how pod make calls to the VM - if L3 networking is not available.

And what limitations - statefull sets, etc. It's not only performance.

Reply:
A concept document explaining how things work would be of great value. That type of content does not belong in the `setup/install` tree.

Would you be willing to write a concept document on this topic to live in the `content` dir? If so, feel free to use your tool of choice and the documentation wg can get it into the tree as time permits.

Reply:
Now all svcs in k8s cluster must be exposed via gateway otherwise, VM can not access them. This also the way our sample works.

Reply:
A VM can actually access any service within a service mesh pod network, if the pod network is accessible via L3, assuming it knows the cluster ID's IP address as per this screenshot:
<img width="865" alt="Screen Shot 2020-05-12 at 10 11 19 PM" src="https://user-images.githubusercontent.com/755849/81773626-96dc0780-949d-11ea-8ab2-a4780145d2b2.png">

This is covered in the examples, not the install however, so we can take this discussion to that review which is here: https://github.com/istio/istio.io/pull/7267